### PR TITLE
Implement experiment health indicators

### PR DESCRIPTION
**Pull request by jessie-ross**
This PR implements experiment health indicators: taking stats, making them user friendly and adds indications. 

In a subsequent PR it will look like this:
Screen Shot 2021-03-24 at 2 20 29 am

Part of #419 

## How has this been tested?

- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)

### CODE REVIEW

**Comment by yanirs** [comment]
I need to step out for a couple of hours, but I left a few suggestions in #481. I'll finish reviewing when I get back.

File: src/lib/analyses.ts [hunk 1]
```diff
@@ -237,3 +239,171 @@ export function getExperimentHealthStats(
...
+    link?: string
+    indicationBrackets: Array<IndicationBracket>
+  }
+
+  const indicatorDefinitions: IndicatorDefinition[] = [
+    {
+      name: 'Assignment distribution matching allocated',
+      value: minVariationProbabilities.assignedDistributionMatchingAllocated,
+      unit: HealthIndicatorUnit.Pvalue,
+      link: '',
```
Comment (yanirs) [inline src/lib/analyses.ts:305]: Why do we set the links to empty strings? Can't we just omit them since `link` is optional?

Comment (jessie-ross) [reply src/lib/analyses.ts:305]: Ah I was going to address this in the next PR

Comment (yanirs) [reply src/lib/analyses.ts:305]: In general, it's usually better to leave out unused fields. :slightly_smiling_face:

**Review by yanirs — APPROVED** [review]
Approving to unblock this since I'll be AFK for the rest of the week. We can discuss my comments and tweaks on #481 next week.

**Pull request merged**